sbi/simulators/simutils.py

```diff
@@ -1,22 +1,25 @@
 from __future__ import annotations
 
-from typing import Callable, Optional
+from typing import Callable, Optional, Tuple
 
 import torch
 from torch import Tensor
 from tqdm.auto import tqdm
-from sbi.simulators.mp_simulator import simulate_mp
+from sbi.simulators.mp_simutils import simulate_mp
 
 
-def simulate_mp_in_batches(
+def simulate_in_batches_mp(
     simulator: Callable,
     theta: Tensor,
     sim_batch_size: Optional[int],
     number_of_workers: int = 4,
     worker_batch_size: int = 20,
     show_progressbar: Optional[bool] = True,
-):
+) -> Tuple[Tensor, Tensor]:
     """
+    Return parameters $\theta$ and corresponding data $x$ simulated using
```

Comment:
Ideally the 1st sentence of the docstring should fit in one line. It's hard sometimes, but quite a few tools 'depend' on it (e.g. for onhover tooltips and the like). You can be as precise as necessary in further lines below.